src/index.html

```diff
@@ -1,12 +1,359 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="/images/logo.png" alt="Logo">
+      </a>
+      <nav class="header__navigation navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item"><a href="#apple" class="is-active">Apple</a></li>
+          <li class="navigation__item"><a href="#samsung">Samsung</a></li>
+          <li class="navigation__item"><a href="#smartphones">Smartphones</a></li>
+          <li class="navigation__item">
+            <a href="#laptops_computers" data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="navigation__item"><a href="#gadgets">Gadgets</a></li>
+          <li class="navigation__item"><a href="#tablets">Tablets</a></li>
+          <li class="navigation__item"><a href="#photo">Photo</a></li>
+          <li class="navigation__item"><a href="#video">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock" height="134px"
+          >
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">
+            Product code: 195434
+          </div>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="price-box card__price-box">
+            <div class="price-box__price">
+              Price:
+            </div>
+            <div class="price-box__amount">
+              $2,199
+            </div>
+          </div>
```

Comment:
How about this:
```suggestion
          <div class="card__price-box">
            <div class="card__price-text">
              Price:
            </div>
            <div class="card__price-amount">
              $2,199
            </div>
          </div>
```

Reply:
In that case text & amount will also be elements of card and it can't be used as separate block. Also in scss file positioning of card__price-text and card__price-amount won't be so clear as in actual solution. There will be two elements, that have to be positioned in other element and that element will be positioned in block. Should I change this or leave actual solution?